src/kubeobjects/serviceentry.go

```diff
@@ -1,4 +1,4 @@
-package istio
+package kubeobjects
```

Comment:
this was good where it was,
a `ServiceEntry` is an isito specific thingy, should stay "locked away" in the `istio` package 😛